README.md

```diff
@@ -8,3 +8,8 @@ and some list items
   * first
   * second
   * third
+  
+  | this      | is | a    | table |
+  |:----------|:---|:-----|:------|
+  | something | in | here | too   |
```

Comment:
This is also fascinating

Reply:
ok, got those. somehow i think i can make another change and then update the pull request... will try...

ta

Reply:
If you make another commit on this branch and push it then it should update the pull request. At least that's how it works on our system (which isn't Github)

Reply:
just done that. hope it worked

Reply:
but now it says i want to merge 2 commits. i thought it would be just one.

Reply:
ah, no. getting mixed up.. yes, should be 2 commits.

Reply:
You might be able to amend the previous commit somehow but I'd make a new one if you want further changes

Reply:
i made another commit (and push to get it to github) and github automatically updated the same pull request.